README.md

```diff
@@ -56,6 +56,15 @@ graphiql interface.
 cargo run schema
 ```
 
+> [!NOTE]
+> Within the schema, 'instrument' can be thought of as equivalent to 'beamline'
+> in most other contexts. It is used to include other facilities such as lab
```

Comment:
'facilities' feels like the wrong term to use here but I can't think of a better one

Reply:
I can't think of a better one either off the top of my head and it is better than nothing. I'd say merge it.

Reply:
"better than nothing" is what I aspire to